app/main.py

```diff
@@ -1,3 +1,46 @@
+from __future__ import division
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __add__(self, other: Any) -> Any:
+        if not isinstance(other, Distance):
+            return Distance(self.km + other)
+        return Distance(self.km + other.km)
+
+    def __iadd__(self, other: Any) -> Any:
+        if not isinstance(other, Distance):
+            self.km += other
+            return self
+        self.km += other.km
+        return self
+
+    def __mul__(self, other: Any) -> Any:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: Any) -> Any:
+        return Distance(round((self.km / other), 2))
+
+    def __lt__(self, other: Any) -> bool:
+        return self.km < other
+
+    def __gt__(self, other: Any) -> bool:
+        return self.km > other
+
+    def __eq__(self, other: Any) -> bool:
+        return self.km == other
```

Comment:
also worth adding a check 
`isinstance(other, Distance)`

here and other places